src/databricks/labs/ucx/source_code/base.py

```diff
@@ -242,3 +246,50 @@ def _make_tree(self) -> Tree:
         if self._tree is None:
             self._tree = Tree.new_module()
         return self._tree
+
```

Comment:
moved here to avoid cyclic imports between cells.py, loaders.py, sources.py and files.py